atlasdb-ete-tests/docker-compose.single-client-with-embedded-and-oracle.yml

```diff
@@ -2,9 +2,12 @@ version: '2'
 
 services:
   oracle:
-    image: palantirtechnologies/oracle-atlasdb:19.20.0.0-se
+    image: palantirtechnologies/oracle-atlasdb:19.20.0.0-test
```

Comment:
Before I approve, we need to create a proper release without the test suffix from oracle-k8s

Reply:
Updated